site/content/blog/2020-11-01-whats-new-in-svelte-november-2020.md

```diff
@@ -0,0 +1,43 @@
+---
+title: What's new in Svelte: November 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+BLURB SUMMARIZING WHAT'S NEW...
+"What's new in Svelte" series! We'll try to make this a monthly blog post in which you'll find out about new features, bug fixes, and a showcase of Svelte projects from around the community.
+
+## New features
+1. 
+
+**What's going on in Sapper?**
+Any updates here?
+
+
+## Impactful bug fixes
+- 
+
+
+## Coming up
+- Rich Harris demoed the possible future of local Svelte development in a talk titled "Futuristic Web Development" at last month's [Svelte Summit](https://sveltesummit.com/). If you're interested in built-in Server-Side Rendering, modern Hot Module Reloading, or how slots make consistent styling easy across pages, [check out the video](https://www.youtube.com/watch?v=qSfdtmcZ4d0).
```

Comment:
Here's my suggested change:

```suggestion
- Rich Harris demoed the possible future of Svelte development in a talk titled "Futuristic Web Development" at last month's [Svelte Summit](https://sveltesummit.com/). The not-yet-public project is called SvelteKit (name may change) and will bring exciting new features to Svelte such as development with out-of-the-box Hot Module Reloading, optional but better Server-Side Rendering and overall more flexibility for build outputs. It will also solve the "Svelte or Sapper?"-question: Sapper will be deprecated and integrated into this project. The transition will be made as smooth as possible - many of Svelte's core team members have large Sapper apps in production, too. If you want to get the full sneak-peek, [check out the video](https://www.youtube.com/watch?v=qSfdtmcZ4d0).
```

Reply:
I'm not sure I'd put "better SSR" here.

I think something like:

>"... and will bring a first-class developer experience and more flexibility for build outputs."

The rest looks fine to me.